src/Common/src/Interop/ComCtl32/Interop.TvItemFlags.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class ComCtl32
+    {
+        [Flags]
+        public enum TvItemFlags : uint
```

Comment:
```suggestion
        public enum TreeViewItemFlags : uint
```

Whilst `Flags` suffix shouldn't be used, looking at [the source](https://github.com/tpn/winsdk-10/blob/master/Include/10.0.14393.0/um/CommCtrl.h#L4835-L4849) the original prefix is `TVIF_`, so I think we can make an exception

Reply:
Also its certainly not `TreeViewItems` 😄
So you kinda need a suffix and can't just get rid of it

Reply:
> Also its certainly not `TreeViewItems`

Please explain.

These are flags that indicate which of structure members contain valid data. Structure can used with the `TVM_GETITEM` or `TVM_SETITEM` messages, both of which deal with *tree-view item's attributes*.
https://docs.microsoft.com/en-us/windows/win32/api/commctrl/ns-commctrl-tvitema

Reply:
TreeViewItem(s) are the items in a tree view. The enum we are talking about is for _attributes of a tree view item_ - you just quoted that yourself. An attribute of something is not the same as the thing itself. Its like the difference between a class and is members. You wouldn't identify a property on a class with the class itself.

Reply:
I prefer using `TVIF` as the name here. Yes, it's ugly, but this is interop and it more closely matches the actual headers. You'd be referring to things as `TVIF.INTEGRAL`, etc. Which is much closer to `TVIF_INTEGRAL` and aligns with the original intent- `TVIF_` was meant to "group" the related constants when there originally was no other way to do so. I've started doing this in other places and I think it holds up pretty well.

Stripping the `TVIF_` in the names makes it hard to tie this back to the original code. If you know the `TVIF.INTEGRAL` pattern you can quickly search for `TVIF_INTEGRAL`. Optionally we can keep the full string, but it makes it significantly harder to read the consuming code.

The other option is to make this C#'ish, but I don't think it makes sense to do it "halfsies". In my WInterop project I went all-in on C#ness and put the original strings in the defines to make writing the code "modern" throughout. I don't think we should do that here, however.

Reply:
We have several different styles right now all caps with and without underscores, CamelCased and all caps enums and a mixed bag of enum member names...
Different refactors from the same contributors appear to be doing different things. 

I am primarily concerned with all the inconsistencies we now have, that add to a mental and a congnitive overloads other contributors now face.

Reply:
> I am primarily concerned with all the inconsistencies we now have, that add to a mental and a congnitive overloads other contributors now face.

As am I. I want these all to be taking off the "prefix" as the name. I'll update the others to match. It is the best option for keeping the defines close to the native code and still grouping. I've tried a number of different approaches over the past several years and I'm convinced this is the best option for this codebase as common as the interop calls are.